src/main/java/nextstep/auth/ui/AuthenticationPrincipalArgumentResolver.java

```diff
@@ -24,8 +24,8 @@ public boolean supportsParameter(MethodParameter parameter) {
     @Override
     public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) throws Exception {
         String authorization = webRequest.getHeader("Authorization");
-        if (!"bearer".equalsIgnoreCase(authorization.split(" ")[0])) {
-            throw new AuthenticationException();
+        if (authorization == null || !"bearer".equalsIgnoreCase(authorization.split(" ")[0])) {
+            return null;
```

Comment:
토큰이 없는 경우 익명 사용자 객체를 리턴하면 어떨까요?
null을 대응하기 보다 객체를 다루는게 좋을 것 같아요
```suggestion
        if (authorization == null || !"bearer".equalsIgnoreCase(authorization.split(" ")[0])) {
            return NonLoginMember();
```